include/hw_unique_key.rst

```diff
@@ -0,0 +1,81 @@
+.. _lib_hw_unique_key:
+
+Hardware unique key
+###################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The hardware unique key library manages symmetric root keys on devices with the Arm CryptoCell security subsystem.
+
+Functionality
+*************
+
+Hardware unique keys (HUKs) are device-specific keys that can be used with functions for key derivation to generate other keys.
+The generated keys can then be used by applications for symmetric encryption or authentication.
+By using HUKs, you can let your application use multiple keys without having to store them on the device, as they can be recreated from a HUK when needed.
+
+.. note::
+   You should use hardware unique keys only for key derivation, but never directly for symmetric encryption or authentication.
+
+The library supports up to 3 different HUKs:
```

Comment:
"Supports up to 3 different HUKs" is probably not specific enough. 

What about something like:

"Supports three different kinds of HUK keys, depending on device type":

Reply:
fixed